index.html

```diff
@@ -0,0 +1 @@
+<p>메인입니다...</p>
```

Comment:
표기가 맞지 않는 거 같습니다. 수정바랍니다.
```suggestion
<p>메인입니다...1232312313</p>
```